roles/accprovision/tasks/process.yaml

```diff
@@ -6,6 +6,9 @@
     name: acc-provision-config
     namespace: "{{ lookup('env', 'WATCH_NAMESPACE') }}"
   register: cmap_data
+  until: cmap_data is not failed
```

Comment:
The file is removed in this commit: https://github.com/noironetworks/acc-provision-operator/pull/8/files

Reply:
yes we can merge one and then I can post rebase for the other